docs/tools/buildkit.md

```diff
@@ -75,12 +75,12 @@ You may install buildkit in other environments. The main pre-requisites are:
 
 * Linux or OS X
 * Git
-* PHP 5.3+ (Extensions: `bcmath curl gd gettext imap intl imagick json mbstring mcrypt openssl pdo_mysql phar posix soap zip`)
-* NodeJS (v5 recommended)
+* PHP 7.0+ (Extensions: `bcmath curl gd gettext imap intl imagick json mbstring mcrypt openssl pdo_mysql phar posix soap zip`)
+* NodeJS (v8 recommended)
 * NPM
 * Recommended (_for [amp](https://github.com/totten/amp) and [civibuild](/tools/civibuild.md)_)
     * Apache 2.2 or 2.4 (Modules: `mod_rewrite`. On SUSE, possibly `mod_access_compat`. This list may not be exhaustive.)
-    * MySQL 5.1+ (client and server)
+    * MySQL 5.6+ (client and server)
```

Comment:
@agh1 FWIW, the environments we actively test are [bknix-min](https://github.com/totten/bknix/blob/master/profiles/min/default.nix#L15-L19) (PHP 5.6, MySQL 5.5, NodeJS 8) and [bknix-max](https://github.com/totten/bknix/blob/master/profiles/max/default.nix#L14-L19) (PHP 7.2, MySQL 5.7, NodeJS 8).

There's obviously discussion about mainline branches going up to PHP 7 as a requirement, but even when that happens, there ought be lag between updating `civicrm-core` reqs and `buildkit` reqs. (Folks continue working on backports+extensions even after mainline changes.)

I'd keep the minimums in here aligned with bknix-min (PHP 5.6, MySQL 5.5, NodeJS 8), but the recommendations can be more aggressive.